rcl/test/rcl/test_time.cpp

```diff
@@ -484,6 +505,29 @@ TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), rcl_time_clock_change_callbacks) {
   reset_callback_triggers();
 }
 
+TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), rcl_time_fail_set_jump_callbacks) {
+  rcl_clock_t fail_clock;
```

Comment:
@Blast545 I'd rather initialize this as `RCL_CLOCK_UNINITIALIZED`, otherwise its content is completely unknown.